### DISCUSSION TITLE
Pytorch cross attention VS Xformers cross attention: What are the differences?

### DISCUSSION THREAD

**Discussion by LukeG89** [comfyanonymous/ComfyUI#5112] [Q&A]
Hi everyone,
Could someone tell me what is the difference between **pytorch cross attention** and **xformers cross attention**?
What are the pros and cons of both?
And why did you choose one rather than the other?

**Comment by LukeG89** [comment]
I guess nobody knows...

**Comment by ltdrdata** [comment]
While the performance is the same, occasionally some custom nodes' dependencies require xformers.
